images/libguestfs/werf.inc.yaml

```diff
@@ -1,25 +1,32 @@
 ---
+# libguestfs-appliance from kubevirt/WORKSPACE
+{{- $libguestfsApplianceTar := "https://storage.googleapis.com/kubevirt-prow/devel/release/kubevirt/libguestfs-appliance/libguestfs-appliance-1.48.4-qcow2-linux-5.14.0-183-centos9.tar.xz" }}
+
 image: {{ $.ImageName }}
 fromImage: base-alt-p11
 import:
 - image: virt-artifact
-  add: /images/kubevirt/{{ $.ImageName }}:latest/usr/local/lib/guestfs/appliance
+  add: /kubevirt/cmd/{{ $.ImageName }}/
+  to: /
   includePaths:
-  - '*'
-  to: /usr/local/lib/guestfs/appliance
+  - entrypoint.sh
   before: setup
 - image: virt-artifact
-  add: /images/kubevirt/{{ $.ImageName }}:latest
+  add: /kubevirt-config-files/
+  to: /
   includePaths:
   - .version
-  - entrypoint.sh
-  to: /
   before: setup
 shell:
+  beforeInstall:
+  - apt-get update && apt-get install --yes curl
+  - mkdir -p /usr/local/lib/guestfs/
+  - curl -L {{ $libguestfsApplianceTar }} | tar -xJ -C /usr/local/lib/guestfs/
+  - apt-get remove curl --yes
```

Comment:
Append `apt clean` and `rm -rf` commands to reduce stage image size.

Reply:
Fixed